frontend/src/components/App/ButtonConfig/index.tsx

```diff
@@ -82,7 +83,7 @@ const ButtonConfig = forwardRef(({ unlabeled }: Props, ref) => {
         (devices.audioInputDevices.length > 0 ||
           devices.audioOutputDevices.length > 0 ||
           devices.videoDevices.length > 0) && (
-          <Selector>
+          <Selector $top={selectorPosition === 'top'} $bottom={selectorPosition === 'bottom'}>
```

Comment:
Attributes should not be preceded with "$" sign, we don't use this pattern in other components

Reply:
Right in this case we souldn't use it thank you. We had a case in a form that we had to use the $... format to avoid writing it into the dom.